streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KTable.scala

```diff
@@ -159,6 +159,16 @@ class KTable[K, V](val inner: KTableJ[K, V]) {
   def toStream[KR](mapper: (K, V) => KR): KStream[KR, V] =
     inner.toStream[KR](mapper.asKeyValueMapper)
 
+  /**
+    * Suppress some updates from this changelog stream, determined by the supplied {@link Suppressed} configuration.
```

Comment:
`@link` is Javadoc, not valid Scaladoc. In scaladoc, it's `[[Suppressed]]`:

```suggestion
    * Suppress some updates from this changelog stream, determined by the supplied [[Suppressed]] configuration.
```